docs/source/posts/2024/2024-08-15_week10.rst

```diff
@@ -0,0 +1,39 @@
+Week 10: Learning GraphQL
+=========================
+
+.. post:: Aug 16 2024
```

Comment:
do not forget to update the filename when updating the date

Reply:
oops, how did I miss that...